### DISCUSSION TITLE
Is it possible to stop prettier from formatting numbers or in general keys and values in a JSON file?

### DISCUSSION THREAD

**Discussion by nbro10** [prettier/prettier#16766] [General]
I have JSON files, which contain very specific values, which I don't want prettier to format. For example, prettier would format `28.50` to `28.5` - please, don't tell me these are equivalent so there's no danger. I want to stop prettier from doing this. I tried to use `json-stringify`, but the numbers are still formatted

```
    "prettier": {
        "overrides": [
            {
                "files": "*.json",
                "options": {
                    "parser": "json-stringify"
                }
            }
        ]
    }
```

I want to use prettier only to format the indentation, blank lines and linebreaks and I don't want it to touch the actual contents. Is it possible to do this? I didn't yet find an option to do this, but this should surely exist.

**Comment by fisker** [comment]
Currently, there is no way to disable it, but I considered removing this functionality, can you open an issue?

**Reply by nbro10** [reply]
Sure, I can open an issue. Thanks. It would be great if you could implement this asap, if it's not too difficult, of course. Cheers

**Reply by nbro10** [reply]
https://github.com/prettier/prettier/issues/16767

**Reply by fisker** [reply]
It's not going to change in near future, we need discuss first.
